### ISSUE TITLE
Pull forge into Grand-Challenge repo

### ISSUE THREAD

**Issue by chrisvanrun** [DIAGNijmegen/rse-grand-challenge-forge#70]
Continued discussion from:
- https://github.com/DIAGNijmegen/rse-grand-challenge-forge/pull/67

The algorithm template generation can be done inline. 

There is one thing I did like about running Forge by itself: seeing deprecation warnings in the Docker build and run. Not sure how we could keep this when pulling it into grand-challenge itself.

**Comment by jmsmkn** [comment]
>  seeing deprecation warnings in the Docker build and run

Couldn't we run that in CI?

**Comment by chrisvanrun** [comment]
> > seeing deprecation warnings in the Docker build and run
> 
> Couldn't we run that in CI?

Yeah we could. Albeit imho we should make it err out on those warnings? Perhaps that is not something we want within the GC CI.

One caveat is that the GPU interaction is something we cannot test for CI: tests skip it for now.

I forsee it roughly as follows:

- [ ] Port code base to GC, plain copy and paste and change some GC pointers
- Subsequent steps:
  - [ ] Disable quality control test by defaults, refactor them in pure CI tests
  - [ ] Enable direct generation to ZIP file-like object to speed up generation


Should we keep the GC JSON generation as descriptive dictionary for the forge, or replace the generation with a singular function that takes a queryset or a pointer to the object to template for?

**Comment by jmsmkn** [comment]
It's like the python warnings we have - we know we have them, monitor them and periodically fix them (especially during upgrades when they're relevant). 

Rather than copy and paste you can move everything over with git, maintaining the history in the PR.

Don't refactor too many things at once, sending over the JSON and keeping the CLI around for now is still fine.

**Comment by chrisvanrun** [comment]
Note: the QC steps have been removed.

**Comment by chrisvanrun** [comment]
Zip file generation is now the default behaviour.

In theory, forge is now ready to be pulled into gc repo.

**Comment by jmsmkn** [comment]
Done in https://github.com/DIAGNijmegen/grand-challenge/commit/a041d8af046481e2829540408ae14965c957ce06
